### PR TITLE
Readd some of the publicly used methods in AbstractBlockSettingsAccessor

### LINKED ISSUE #9: quilted fabric object builder api missing getMaterial()
[issue with eden ring](https://github.com/paulevsGitch/EdenRing/issues/27)
missing `net.fabricmc.fabric.mixin.object.builder.AbstractBlockSettingsAccessor.getMaterial()`

### PR DESCRIPTION
**Pull request by triphora**
fixes #9

### CODE REVIEW

File: fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/mixin/object/builder/AbstractBlockSettingsAccessor.java [hunk 1]
```diff
@@ -27,12 +27,73 @@
...
+	@Accessor
+	boolean getOpaque();
+
+	@Deprecated
+	@Accessor
+	boolean getIsAir();
+
+	@Deprecated
+	@Accessor
+	boolean isToolRequired();
```
Comment (EnnuiL) [inline fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/mixin/object/builder/AbstractBlockSettingsAccessor.java:91]: A tiny nitpick here:
```suggestion
	boolean isToolRequired();

```

**Review by EnnuiL — CHANGES_REQUESTED** [review]
Other than this really tiny style nitpick, I believe this is ready

File: fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/mixin/object/builder/AbstractBlockSettingsAccessor.java (hunk 1 above)
Comment (triphora) [reply fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/mixin/object/builder/AbstractBlockSettingsAccessor.java:91]: "Suggestion cannot be identical to original text"

Comment (EnnuiL) [reply fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/mixin/object/builder/AbstractBlockSettingsAccessor.java:91]: huh, I guess it thought the new line was an unnecessary whitespace for some reason; Welp, I guess it'll have to be done manually then

**Review by EnnuiL — APPROVED** [review]

**Pull request merged**